framework/src/positions/ElementNodePositions.C

```diff
@@ -0,0 +1,88 @@
+//* This file is part of the MOOSE framework
+//* https://mooseframework.inl.gov
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "ElementNodePositions.h"
+
+#include "libmesh/parallel_algebra.h"
+
+registerMooseObject("MooseApp", ElementNodePositions);
+
+InputParameters
+ElementNodePositions::validParams()
+{
+  InputParameters params = Positions::validParams();
+  params.addClassDescription("Positions of element nodes.");
+  params += BlockRestrictable::validParams();
+
+  // Element nodes must be sorted to remove duplicates.
+  params.set<bool>("auto_sort") = true;
```

Comment:
actually let s suppress this too. Since we overrode finalize() and did not use the boolean, this parameter is now inactive